main.rb

```diff
@@ -0,0 +1,44 @@
+require './app'
+
+class Main
+  def initialize
+    @app = App.new
+    @output = 0
+  end
+
+  def options
+    print 'WELCOME TO THE SCHOOL LIBRARY APP'
+    while @output != 7
+      print "\nPlease choose an option by entering a number \n\n"
+      print "1 - List all books \n"
+      print "2 - List all people \n"
+      print "3 - Create a person (teacher or student) \n"
+      print "4 - Create a book \n"
+      print "5 - Create a rental \n"
+      print "6 - List all rentals for a person \n"
```

Comment:
- [ ] [OPTIONAL] Kindly consider printing the id of the person, so when you want to list the rentals the user can know the id of the person, as for now there is no way to get the person id to list the rentals